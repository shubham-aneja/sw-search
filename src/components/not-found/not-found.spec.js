import React from 'react';
import { mount } from 'enzyme';
import NotFound from './index.js'

describe('NotFound', () => {
    it('rendering', () => {
        const app = mount(<NotFound />)
        expect(app.html()).toMatchSnapshot();

    });

});

