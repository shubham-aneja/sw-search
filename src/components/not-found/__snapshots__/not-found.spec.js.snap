// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound rendering 1`] = `"<div class=\\"Home-app\\"><h2>404 Not Found</h2></div>"`;
